IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/include/user_settings.h

```diff
@@ -445,8 +837,11 @@ See wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h for details on debug options
 #define WOLFSSL_TEST_STRAY 1
 #define USE_ESP_DPORT_ACCESS_READ_BUFFER
 #define WOLFSSL_ESP32_HW_LOCK_DEBUG
+#define WOLFSSL_DEBUG_MUTEX
 #define WOLFSSL_DEBUG_ESP_RSA_MULM_BITS
```

Comment:
Does it make sense to enable debug gates in the example config?  Maybe so.  Just wondering if you considered the overhead.

Edit:  There's actually another `#define WOLFSSL_ESP32_HW_LOCK_DEBUG` below, commented out.

Reply:
> Does it make sense to enable debug gates in the example config? Maybe so. 

Maybe, but not much interesting in this template example. I'll review with the other examples & see if any make sense without being too verbose. There's already some informative "always on" messages. See the `ESP_LOGI()` in the [TLS Client Example](https://github.com/wolfSSL/wolfssl/blob/master/IDE/Espressif/ESP-IDF/examples/wolfssl_client/main/client-tls.c). I'll definitely be adding more settings to control these from Kconfig / menuconfig.

> Another #define WOLFSSL_ESP32_HW_LOCK_DEBUG below, commented out.

Good catch. I've removed the extra one.